scripts/programming_files.py

```diff
@@ -222,10 +255,11 @@ def format_df_for_d878(df: pd.DataFrame) -> pd.DataFrame:
     df_878.loc[is_dmr, "Color Code"] = dmr_codes["color"]
     df_878.loc[is_dmr, "Slot"] = dmr_codes["slot"]
 
-    # TODO: Need to programatically generate a "scan list" csv file in order to be able
-    # to scan all the frequencies in the Roundabout.  For now - a dummy file added to
-    # ./assets.
-    df_878["Scan List"] = "Roundabout"
+    is_metro = df.apply(lambda x: distance_between(x["Coordinates"], DOWNTOWN_SEATTLE) < METRO_DISTANCE, axis=1)
+    df_878.loc[is_metro, "Scan List"] = "Metro"
+    is_north = df.apply(lambda x: x["Coordinates"][0] > DOWNTOWN_SEATTLE[0], axis=1)
+    df_878.loc[~is_metro & is_north, "Scan List"] = "North"
+    df_878.loc[~is_metro & ~is_north, "Scan List"] = "South"
```

Comment:
I think this is a good solution for the current setup. We'll have a few repeaters east and west of Seattle labeled North and South, but a more general clustering would take a lot of effort for arguably no better results ! The groups are also fairly well balanced, which is great.